docs/source/common/trainer.rst

```diff
@@ -128,7 +128,9 @@ So you can run it like so:
 
     if __name__ == '__main__':
         parser = ArgumentParser()
-        parser = Trainer.add_argparse_args(parser)
+        parser = Trainer.add_argparse_args(
+            parser.add_argument_group(title="pl.Trainer args")
```

Comment:
```suggestion
            # Group the Trainer arguments together
            parser.add_argument_group(title="pl.Trainer args")
```

Reply:
Done in 5526b42d. Changed casing to match other comments in the file (e.g. `# print full summary of all modules and submodules`)